feat: community events page (#1120)
asyncapi/website: pages/community/events/community-call.md

---
title: 'AsyncAPI Community Call'
type: COMMUNITY CALL

> description: 'Join us for our upcoming community call to see what’s on our roadmap, early feature demos and product updates from our engineering team.

Reviewer: ```suggestion
description: 'Join our upcoming community call to discuss our roadmap, feature demos, engineering updates, and more.'
```

It's not really a product, right? it's OSS :D 

and similarly, not really a team 😄, it's a community with engineering contributors

'
weight: 1
cover: https://cdn.dribbble.com/users/499731/screenshots/6648311/viceland_assets.png
date: 2022-10-04T06:00:00+01:00
---

> Join us for our upcoming community call to see what’s on our roadmap, early feature demos and product updates from our engineering team.

Reviewer: ```suggestion
Join our upcoming community call to discuss our roadmap, feature demos, engineering updates, and more.
```

> ### What you will learn

Reviewer: ```suggestion

```

hmm nope, this header doesn't work... i'd rather remove it unless you have a specific goal for a new section there
